File: Objetivo 1/Exemplo1_Java_MVC/src/model/Usuario.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package model;

/**
 *
 * @author vpsil
 */
public class Usuario {
    public String nome;
    public String sobrenome;
    public String email;
    public String senha;
    
    public Usuario(String nome,String sobrenome,String email,String senha) {
        this.nome = nome;
        this.sobrenome = sobrenome;
        this.email = email;
        this.senha = senha;
    }

    @Override
    public String toString() {
        return "Usuario{" + "nome=" + nome + ", sobrenome=" + sobrenome + ", email=" + email + ", senha=" + senha + '}';
    }
}

File: Objetivo 1/Exemplo1_PHP_MVC/model/Usuario.php
<?php

/**
 * Description of Usuario
 *
 * @author vpsil
 */
class Usuario {
    
    public $nome;
    public $sobrenome;
    public $email;
    public $senha;
    
    public function __construct($nome, $sobrenome, $email, $senha) {
        $this->nome = $nome;
        $this->sobrenome = $sobrenome;
        $this->email = $email;
        $this->senha = $senha;
    }
    
    public function __toString() {
        return "Usuario[nome=$this->nome, sobrenome=$this->sobrenome, email=$this->email, senha=$this->senha]";
    }

}
